src/UserAccount.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const UserAccount = ({ userId }) => {
+  const [user, setUser] = useState(null);
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      try {
+        const response = await axios.get(`/api/user/${userId}`);
```

Comment:
These axios calls will currently fail since these aren't the right URLs. For now just make a mock user object at the top of the file to consume for visually displaying